test/list_package_dirs/lockfile_error_test.dart

```diff
@@ -19,7 +19,7 @@ void main() {
       'list-package-dirs',
       '--format=json'
     ], outputJson: {
-      'error': contains('The lockfile must be a YAML mapping.'),
+      'error': contains('Expected a YAML mapping.'),
```

Comment:
Should we not say "lockfile" in the error message anymore? or is it u necessary because we're using source_span?

Reply:
I don't think it is needed when we have the file name from the source span